### ISSUE TITLE
Update base64url to fix vunerability

### ISSUE THREAD

**Issue by mir4ef** [AzureAD/passport-azure-ad#383]
There is a known vulnerability in [base64url 2.0.0](https://nodesecurity.io/advisories/664). Could you please update your dependency to a later version that has patched the vulnerability? We are using `nsp` as part of our build process and our builds are failing due to this.

```

┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Out-of-bounds Read                                                 │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ base64url                                                          │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 7.1 (High)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 2.0.0                                                              │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <3.0.0                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ >=3.0.0                                                            │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ myapp@0.0.1 > passport-azure-ad@3.0.12 >                │
│            │ base64url@2.0.0                                                    │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/658                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

**Comment by leoschweizer** [comment]
See #382

**Comment by nehaagrawal** [comment]
I have released 4.0.0. Please check. Closing this issue for now.
